src/AppInstallerCLICore/Workflows/ArchiveFlow.cpp

```diff
@@ -4,27 +4,33 @@
 #include "ArchiveFlow.h"
 #include "winget/Archive.h"
 #include "winget/Filesystem.h"
+#include "PortableFlow.h"
 
 using namespace AppInstaller::Manifest;
 
 namespace AppInstaller::CLI::Workflow
 {
+    namespace
+    {
+        constexpr std::wstring_view s_Extracted = L"extracted";
+    }
+
     void ExtractFilesFromArchive(Execution::Context& context)
     {
         const auto& installerPath = context.Get<Execution::Data::InstallerPath>();
-        const auto& installerParentPath = installerPath.parent_path();
+        std::filesystem::path destinationFolder = installerPath.parent_path() / s_Extracted;
+        std::filesystem::create_directory(destinationFolder);
 
-        // TODO: For portables, extract portables to final install location and log to local database.
-        HRESULT hr = AppInstaller::Archive::TryExtractArchive(installerPath, installerParentPath);
-        AICLI_LOG(CLI, Info, << "Extracting archive to: " << installerParentPath);
+        AICLI_LOG(CLI, Info, << "Extracting archive to: " << destinationFolder);
```

Comment:
In testing this, it feels like perhaps there should be output written to inform the user the archive is being extracted. I noticed with particularly large archives there appeared to be significant delay between `Successfully verified installer hash` and `Starting package install. . .`

Reply:
Agreed; the extraction should be part of the install phase, not the download.

Reply:
We may also be able to have progress callbacks for the extraction, and certainly can for the progress of moving files to their target location.